go/pkg/log/repository/log.go

```diff
@@ -0,0 +1,92 @@
+package repository
+
+import (
+	"context"
+	"errors"
+	log "github.com/chroma-core/chroma/go/database/log/db"
+	"github.com/jackc/pgx/v5"
+)
+
+type LogRepository struct {
+	conn    *pgx.Conn
+	queries *log.Queries
+}
+
+func (r *LogRepository) InsertRecords(ctx context.Context, collectionId string, records [][]byte) (insertCount int64, err error) {
+	var tx pgx.Tx
+	tx, err = r.conn.BeginTx(ctx, pgx.TxOptions{})
+	var collection log.Collection
+	queriesWithTx := r.queries.WithTx(tx)
+	defer func() {
+		if err != nil {
```

Comment:
What is the plan for logging?

Reply:
I don't have logging here yet, but my goal was to do something similar to quota, and decouple logging with error reporting. Happy to chat about it to see if it make sense.